src/jit/compiler.cpp

```diff
@@ -4428,11 +4428,12 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, CORJIT_F
     // Stash the current estimate of the function's size if necessary.
     if (verbose)
     {
-        compSizeEstimate = 0;
+        compSizeEstimate  = 0;
         compCycleEstimate = 0;
         for (BasicBlock* block = fgFirstBB; block != nullptr; block = block->bbNext)
         {
-            for (GenTreeStmt* statement = block->firstStmt(); statement != nullptr; statement = statement->getNextStmt())
+            for (GenTreeStmt* statement = block->firstStmt(); statement != nullptr;
+                 statement              = statement->getNextStmt())
             {
```

Comment:
We can change what jit-format does for assignments and declarations. I've been hearing a lot of people say they don't like how either instance is formatted.